classes/Settings.php

```diff
@@ -452,8 +452,8 @@ public function GetMainClientIP(){
             // TODO the algorithm below just gets the first IP in the list...we might want to make this more intelligent somehow
             $result = $this->GetClientIPs();
             $result = reset($result);
-            $result = isset($result[0]) ? $result[0] : null;
-        } elseif(isset($_SERVER['REMOTE_ADDR'])) {
+            $result = isset($result[0]) ? $result[0] : 'N/A - empty referer';

```

Comment:
@carmelopreeostudios please leave this as null as we discussed - we need to tackle the original problem which needs some more investigation (i.e the IP is coming NULL)